index.html

```diff
@@ -1,182 +1,231 @@
+              </ul>
+              <ul class="facebook">
+                <li class="canopy-li">FACEBOOK</li>
+                <li class="common">Fullstack Dev</li>
+                <li class="common">2015</li>
+              </ul>
             <p class="card-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p class="fcb-p">Experimental content creation feature that allows users to add to an existing story over the course of a day without
+            spamming their friends.</p>
             <ul class="languages-btn">
               <li class="btn"><button>html</button></li>
+              <li class="btn ruby"><button>Ruby on Rails</button></li>
               <li class="btn"><button>css</button></li>
               <li class="btn"><button>javascript</button></li>
             </ul>
             <button type="button" class="big-btn">See Project</button>
+            
           </div>
-          </li>
-  
-        </ul>
-      </section>
-     
-
-      <section class="about-me">
-        <div>
-          <div class="myself">
-        <h5>About <br>Myself</h5>
-        <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <h6 class="connect social-icons">LET`S CONNECT</h6>
-        <address>
-          <ul class="social">
-            <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/Linkedin-icon.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/M.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/angel.svg" alt=""></a></li>
-          </ul>
-        </address>
+        </li>
+
+      </ul>
+    </section>
+
+
+    <section id="About" class="about-me">
+        <div class="myself">
+          <h5>About <br>Myself</h5>
+          <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.</p>
+          <h6 class="connect social-icons">LET`S CONNECT</h6>
+          <address>
+            <ul class="social">
+              <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+              <li><a href="#"><img src="images/Linkedin-icon.svg" alt=""></a></li>
+              <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+              <li><a href="#"><img src="images/M.svg" alt=""></a></li>
+              <li><a href="#"><img src="images/angel.svg" alt=""></a></li>
+            </ul>
+          </address>
           <button class="big-btn about-btn">Get my resume</button>
```

Comment:
- [ ] There is an issue with this button layout/spacing because it's showing at the bottom of the section and there is a big space between this element and the social icons. Please, reduce the space between the social icons and "Get my resume" button to make your portfolio look even better :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144674171-5b46aded-701e-46c5-afe5-efa706ea5679.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144674187-b13a017e-5674-46ec-943b-86bb1e55820d.png)